### DISCUSSION TITLE
Repo Created

### DISCUSSION THREAD

**Discussion by Fuzzworth** [Fuzzworth/alx_test_suite#1] [General]
Checks

The Checker will be released at the end of the project (1-2 days before the deadline). We strongly encourage the entire class to work together to create a suite of checks covering both regular tests and edge cases for each task. See task 8. Test suite.

**Comment by ghost** [comment]
Hello pls dd u delete simple shell test case?

**Reply by Fuzzworth** [reply]
Delete is such a hateful word.... I added a commit to it cause of copyright issues... there is no way however, that I could possibly stop some one from cloning and resetting the commits... *wink wink*
